strawberry_django/fields/filter_types.py

```diff
@@ -82,6 +82,37 @@ class FilterLookup(BaseFilterLookup[T]):
     )
 
 
+@strawberry.input
+class StrFilterLookup(BaseFilterLookup[T]):
```

Comment:
https://github.com/strawberry-graphql/strawberry-django/pull/825

 So there specific filter for `StrFilterLookup` that used for UUID and str. But `FilterLookup[str]` is just for backward compatibility in case if someone implement something with it.